rollup.config.js

```diff
@@ -22,6 +23,11 @@ const defaultPlugins = [
     customCompression: (content) => brotliPromise(Buffer.from(content)),
     fileName: '.br',
   }),
+  replace({
+      'zh-cn': 'zh-hans',
+      'zh-tw': 'zh-hant',
+      preventAssignment: true,
+  })
```

Comment:
IIUC, this amounts to a global search-replace of these strings. That seems exceedingly risky. Why not just do a local replace in the code that passes `LANGUAGE_CODE`?

Reply:
I would agree with that, then we might need to revert https://github.com/inducer/relate/commit/05f95361dd4737ac06ae61e6c3d9284457f227ca#diff-3b26411dfcb9fd30a16787b21f7786d3e24c169058aedae6bc8a3e86efe87ecb back?

Reply:
How about #913?